src/themes/v9/global_styling/variables/_buttons.scss

```diff
@@ -14,11 +14,11 @@ $ouiButtonHeightSmall: $ouiSizeXL !default;
 $ouiButtonHeightXSmall: $ouiSizeL !default;
 
 // sass-lint:disable no-color-literals
-$ouiButtonColorDisabled: lightOrDarkTheme(#AFAFAF, #595F64) !default;
+$ouiButtonColorDisabled: lightOrDarkTheme(#AFAFAF, #444444) !default;
```

Comment:
Can these use?
```suggestion
$ouiButtonColorDisabled: lightOrDarkTheme($ouiColorDarkShade, $ouiColorLightShade) !default;
```

Reply:
Updated comment to use light theme token that matches value (originally had these but not all of them used tokens so revert to literals), but will accept the suggestion.